utils/json.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2019-2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package utils
+
+import (
+	"bufio"
+	"fmt"
+	"strings"
+)
+
+// Update the JSON body if the matching key is found
+// and replace the value.
+// e.g., "whitelisted-subnets" is the key and value is "a,b,c".
+func UpdateJSONKey(jsonBody string, k string, v string) (string, error) {
+	k = fmt.Sprintf(`"%s":`, k)
+	lines := make([]string, 0)
+	scanner := bufio.NewScanner(strings.NewReader(jsonBody))
+	for scanner.Scan() {
+		line := scanner.Text()
+		if !strings.Contains(line, k) {
+			lines = append(lines, line)
+			continue
+		}
+
+		idx := strings.Index(line, k)
+		if idx == -1 {
```

Comment:
error msg refers to fixed whitelisted-subnets field, should refer to k. Besides that, probably the code shall use generic JSON unmarshal + hash update + JSON marshal to avoid assuming something on the input format (I understand this assumes there is one field per line)